### PR TITLE
TELCODOCS-522 - manual 4.10 CP for #49805

### PR DESCRIPTION
**Pull request by aireilly**
https://issues.redhat.com/browse/TELCODOCS-522

manual 4.10 CP for https://github.com/openshift/openshift-docs/pull/49805

Preview: http://file.emea.redhat.com/aireilly/cp-td522-410/scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-clusters-at-scale.html

cherry-pick changes for 4.10:

* MERGED:       _topic_maps/_topic_map.yml
* DELETED:      installing/installing_with_agent_based_installer/installing-with-agent-based-installer.adoc
* RENAMED:      modules/cnf-topology-aware-lifecycle-manager-pao-update.adoc
* MERGED:       modules/cnf-topology-aware-lifecycle-manager-preparing-for-updates.adoc
* DELETED:      modules/sno-du-configuring-performance-addons.adoc
* MERGED:       modules/ztp-acm-adding-images-to-mirror-registry.adoc
* DELETED:      modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc
* DELETED:      modules/ztp-configuring-pgt-compliance-eval-timeouts.adoc
* MERGED:       modules/ztp-configuring-the-cluster-for-a-disconnected-environment.adoc
* MERGED:       modules/ztp-deploying-a-site.adoc
* DELETED:      modules/ztp-generating-ran-policies.adoc
* MERGED:       modules/ztp-manually-install-a-single-managed-cluster.adoc
* MERGED:       modules/ztp-preparing-for-the-gitops-ztp-upgrade.adoc
* MERGED:       modules/ztp-preparing-the-hub-cluster-for-ztp.adoc
* MERGED:       modules/ztp-preparing-the-ztp-git-repository.adoc
* MERGED:       modules/ztp-required-changes-to-the-git-repository.adoc
* MERGED:       modules/ztp-roll-out-the-configuration-changes.adoc
* MERGED:       modules/ztp-using-pgt-to-update-source-crs.adoc
* DELETED:      modules/ztp-ztp-custom-resources.adoc
* DELETED:      scalability_and_performance/cnf-low-latency-tuning.adoc
* DELETED:      scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
* DELETED:      scalability_and_performance/ztp-deploying-disconnected.adoc
* MERGED:       snippets/ztp-operator-groups-namespace.adoc
* MERGED:       snippets/ztp-operator-subs.adoc

### CODE REVIEW

File: modules/ztp-telco-ran-software-versions.adoc [hunk 1]
```diff
@@ -14,14 +14,17 @@ The Red Hat Telco Radio Access Network (RAN) version {product-version} solution
...
+|Hub cluster {product-title} version
+|4.8 or 4.9
+
 |ZTP GitOps plugin
-|4.11
+|4.10
 
 |{rh-rhacm-first}
-|{rh-rhacm-version}
+|2.3 or 2.4
```
Comment (imiller0) [inline modules/ztp-telco-ran-software-versions.adoc:24]: @yliu127 can you check these versions. I think 4.10 was tested with ACM 2.4 and 2.5 and gitops 1.4?

File: modules/ztp-monitoring-policy-deployment-progress.adoc [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+[source,terminal]
+----
+{
+  "lastTransitionTime": "2022-11-09T07:28:09Z",
+  "message": "The ClusterGroupUpgrade CR has upgrade policies that are still non compliant",
+  "reason": "InProgress",
+  "status": "True",
+  "type": "Progressing"
+}
+----
```
Comment (imiller0) [inline modules/ztp-monitoring-policy-deployment-progress.adoc:45]: @jc-rh is this status the same between 4.10 and 4.11? (specifically, is this output accurate for 4.10 TALM?)

Comment (jc-rh) [reply modules/ztp-monitoring-policy-deployment-progress.adoc:45]: No, the Progressing condition is 4.12 only. BTW, the message for this condition is different in 4.12 as well:
https://github.com/openshift-kni/cluster-group-upgrades-operator/blob/a04dc548b4c240c0c7294b5cb3ae9441d9ca4d69/tests/kuttl/tests/upgrade-complete/02-assert.yaml#L26
@aireilly Switch to 4.10/4.11 version of this file as a reference for this PR.

Comment (aireilly) [reply modules/ztp-monitoring-policy-deployment-progress.adoc]: For 4.10, PTAL at: https://github.com/openshift/openshift-docs/pull/53026/commits/84b652c577d3686777b4ec19b6228342acb9f56f

I've review and a seperate PR fix for for 4.11

Comment (aireilly) [reply modules/ztp-monitoring-policy-deployment-progress.adoc]: 4.11: https://github.com/openshift/openshift-docs/pull/53181

File: modules/ztp-telco-ran-software-versions.adoc (hunk 1 above)
Comment (aireilly) [reply modules/ztp-telco-ran-software-versions.adoc:24]: This is what we documented at time of release: https://docs.openshift.com/container-platform/4.10/scalability_and_performance/ztp-deploying-disconnected.html#ztp-preparing-the-hub-cluster-for-ztp_ztp-deploying-disconnected

File: modules/ztp-monitoring-policy-deployment-progress.adoc (hunk 2 above)
Comment (aireilly) [reply modules/ztp-monitoring-policy-deployment-progress.adoc]: @jc-rh ^^

File: modules/ztp-telco-ran-software-versions.adoc (hunk 1 above)
Comment (yliu127) [reply modules/ztp-telco-ran-software-versions.adoc:24]: correct. CC @aireilly, 4.10 is tested with these versions: https://docs.google.com/spreadsheets/d/1ui8Jw9oE9-wCbmU-AqLkiLA4UXyYRoYhUZEt445Zaa4/edit?usp=sharing

**Pull request merged**